src/dkg/round2.rs

```diff
@@ -118,9 +127,66 @@ pub fn import_secret_package(
     SerializableSecretPackage::deserialize_from(&serialized[..]).map(|pkg| pkg.into())
 }
 
+pub fn round2<'a, P, R: RngCore + CryptoRng>(
+    self_identity: &participant::Identity,
+    round1_secret_package: round1::SecretPackage,
```

Comment:
Nit: can we use a `&round1::SecretPackage` instead, to avoid passing large values on the stack?